backend/sqlc/episode-queries.go

```diff
@@ -53,6 +53,7 @@ func (q *Queries) mapToEpisodes(episodes []getEpisodesRow) []common.Episode {
 			SeasonID:              e.SeasonID,
 			AssetID:               e.AssetID,
 			Assets:                assetIDs,
+			AssetVersion:          e.AssetVersion,
```

Comment:
I think it would be slightly more readable if this was `AssetVersion: e.DateUpdated`. I.e. not rename the field in the query

Reply:
I agree. It makes it clear that this is the current mapping.